packages/e2e-test-utils/src/visitSettings.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Creates a new story.
+ */
+async function visitSettings() {
+  await visitAdminPage('index.php');
+
+  const adminMenuItem = await expect(page).toMatchElement(
+    '#menu-posts-web-story'
+  );
+  await page.hover('#menu-posts-web-story a');
+  await expect(page).toMatchElement('#menu-posts-web-story.opensub');
+
+  await expect(adminMenuItem).toClick('a', {
+    text: 'Settings',
+  });
+  await page.waitForNavigation();
+
+  await expect(page).toMatch('Google Analytics Tracking ID');
```

Comment:
Wouldn't it be better to do something like this 

https://github.com/google/web-stories-wp/blob/ace36dc1e8025b0eba5eece43ee58f688463f03e/packages/e2e-tests/src/specs/dashboard/integrations/sitekit.js#L37-L49